src/context-selection/data-set-selector-bar-item/use-data-set.js

```diff
@@ -4,7 +4,6 @@ import { useDataSetId } from '../use-context-selection/index.js'
 const QUERY_DATA_SET = {
     dataSet: {
         resource: 'dataSets',
-        id: ({ id }) => id,
```

Comment:
What happened here?

Reply:
I've removed this whilst sketching as well, to silence an error that occured during the idb serialization. Anything in the query that can't be serialized will throw a datacloneerror.

Reply:
Also, this pattern isn't necessary with RQ, but I've not pushed the changes where I've addressed that yet.